src/README.md

```diff
@@ -0,0 +1,41 @@
+To get the same behaviour as in the main branch, run:
+coupled_heat_equations(iterate=1, values=Dict(:delta_t_min=>1, :t_max=>3600, :delta_t_cpl=>100))
```

Comment:
It would be best if `coupled_heat_equations()` without any arguments produced the same behavior as in the main branch currently (i.e., change the default arguments and adjust the code snippets below). Also, since we merge into the main branch, "To get the same behaviour as in the main branch" will not make sense anymore.